src/request/src/middleware/src/storage/index.js

```diff
@@ -127,8 +127,8 @@ export default class Storage {
 
   removeById(collection, id) {
     return queue.add(() => {
-      if (!isString(id)) {
-        return Promise.reject(new Error('id must be a string', id));
+      if (isString(id) === false) {
```

Comment:
The style guide said to [use shorthands for booleans](https://github.com/Kinvey/style-guides/blob/a6237b72b60bc14c8d5023a24ce9dcf3d8e78937/backend/es6.md#comparison--shortcuts) and I think it's neater.

(This is also the area of the removed second argument to KinveyError's constructor I mentioned in another comment)